compass/Coordinator.java

```diff
@@ -292,10 +293,21 @@ private void start() throws InterruptedException {
           }
         }
 
-        if (!validateTransactionsToApprove(trunk, branch)) {
-          throw new RuntimeException("Trunk & branch were not consistent!!! T: " + trunk + " B: " + branch);
+        try {
+          // This can also throw a RuntimException in case of API Error
+          if (!validateTransactionsToApprove(trunk, branch)) {
+            throw new RuntimeException("Trunk & branch not consistent");
+          }
+        } catch (RuntimeException e) {
+          if (++validationRetries >= config.validationAttempts) {
+            throw new RuntimeException("Trunk & branch were not consistent on multiple attempts!!! T: " + trunk + " B: " + branch);
+          } else {
+            log.warn("Validation failed, #{} of #{} attempts. Trying again...", validationRetries, config.validationAttempts);
+            // Perform gTTA and Validation again
+            continue;
```

Comment:
what was the problem you saw exactly before and how does this solve it?

Reply:
Tips were obtain by gTTA, validated right away against the validator and if they were not found solid a RunTimeException was thrown.
I've discovered that IRI, unlike Hornet, make jota throw an exception if the transaction you are testing for consistency is missing from the DB; hence why I've also accounted for exceptions to be thrown by `validateTransactionsToApprove`.
In both cases this PR will make Compass do gTTA, wait a number of seconds and request validation against all configured validators. If any API or validation fail the entire process (gTTA + delay + validations) is performed all over again, up to a certain amount of retries; if the process goes well once the counter is reset, if the maximum number of attempts is reached Compass will throw the same way as it used to do before after a single failure.